src/library/dlgtagfetcher.cpp

```diff
@@ -582,13 +594,15 @@ void DlgTagFetcher::slotLoadBytesToLabel(const QByteArray& data) {
     coverInfo.setImage();
 
     loadingProgressBar->setVisible(false);
-    statusMessage->setText(tr("Cover art is ready to be applied"));
+    statusMessage->clear();
     statusMessage->setVisible(true);
 
     m_fetchedCoverArtByteArrays = data;
     m_pWFetchedCoverArtLabel->loadData(
             m_fetchedCoverArtByteArrays); // This data loaded because for full size.
     m_pWFetchedCoverArtLabel->setCoverArt(coverInfo, fetchedCoverArtPixmap);
+
+    btnApplyCover->setEnabled(!data.isNull());
```

Comment:
nitpick, but easier to read.
```suggestion
    btnApplyCover->setDisabled(data.isNull());
```